requirements.txt

```diff
@@ -20,12 +20,14 @@ types-pyyaml
 trustme
 cryptography
 coverage
-httpx==0.16.*
+httpx>=0.18.2
 pytest-asyncio==0.14.*
 async_generator; python_version < '3.7'
 
 
 # Documentation
-mkdocs
+mkdocs>=1.2.2
 mkdocs-material
 
+# py3.10 workarounds
+https://github.com/aaugustin/websockets/archive/d5cf1efb737a943583b1e5b4ceca5376bdc3995f.tar.gz#egg=websockets; python_version >= '3.10'
```

Comment:
```suggestion
```

Reply:
@aaugustin the latest websockets is now hanging the uvicorn test suite, and I'm not sure why.